spec/services/set_reject_by_default_spec.rb

```diff
@@ -17,10 +17,10 @@
       ['4 Jan 2019 11:00:00 PM GMT', '2 Mar 2019 0:00:00 AM GMT',  'safely within GMT'],
       ['1 Jul 2020 11:00:00 PM BST', '30 Jul 2020 0:00:00 AM BST', 'during the 20-day summer period'],
       ['21 Nov 2020 12:00:00 PM GMT', '2 Feb 2021 0:00:00 AM GMT', 'near the UCAS winter break'],
-      ['1 Sept 2021 0:00:00 AM BST', '29 Sept 2021 23:59:59 PM BST', 'not beyond the 2021 EoC deadline'],
-      ['7 Sept 2021 0:00:00 AM BST', '1 Oct 2021 23:59:59 PM BST', 'beyond the 2021 EoC deadline'],
-      ['20 Sept 2021 0:00:00 AM BST', '1 Oct 2021 23:59:59 PM BST', 'beyond the 2021 EoC deadline'],
-      ['29 Sept 2021 0:00:00 AM BST', '1 Oct 2021 23:59:59 PM BST', 'beyond the 2021 EoC deadline'],
+      ['1 Sept 2021 0:00:00 AM BST', '29 Sept 2021 23:59:59 PM BST', '7 days before the apply 1 deadline'],
+      ['7 Sept 2021 0:00:00 AM BST', '29 Sept 2021 23:59:59 PM BST', '1 day before apply 1 deadline'],
+      ['20 Sept 2021 0:00:00 AM BST', '29 Sep 2021 23:59:59 PM BST', '1 day before apply 2 deadline'],
+      ['1 Oct 2021 0:00:00 AM BST', '29 Sep 2021 23:59:59 PM BST', '1 day after reject by default cycle timetable date'],
```

Comment:
How could the application be submitted after the deadlines have already passed?

Reply:
Yes - true, it's not possible to submit an application post the Apply 2 deadline - have removed.